options/locale/locale_en-US.ini

```diff
@@ -3394,6 +3394,10 @@ runners.status = Status
 runners.id = ID
 runners.name = Name
 runners.owner_type = Type
+runners.owner_type.unknown = Unknown type: %s
+runners.owner_type.global = Global
+runners.owner_type.organization = Organization
+runners.owner_type.repository = Repository
```

Comment:
I agree with @wxiaoguang, we don't need the 8th translation of "Repository". All of these can be moved to top-level scope.

```
$ rg '= Repository$' options/locale/locale_en-US.ini
51:repository = Repository
1883:settings.options = Repository
2075:settings.event_repository = Repository
2753:packages.repository = Repository
3117:notices.type_1 = Repository
3254:conan.details.repository = Repository
3407:runners.task_list.repository = Repository
```

Reply:
I never thought there are already so many ....

So maybe it's the time to have some globally shared `entity type` or `model type`.

Reply:
Opened https://github.com/go-gitea/gitea/issues/24488 for this topic.